relayer-cli/src/commands/tx/connection.rs

```diff
@@ -22,12 +24,44 @@ pub struct TxRawConnInitCmd {
     #[options(free, help = "identifier of the source connection")]
     src_connection_id: Option<String>,
 
-    #[options(help = "identifier of the destination connection", short = "d")]
+    #[options(free, help = "identifier of the destination connection")]
     dest_connection_id: Option<String>,
+
+    #[options(help = "account sequence of the signer", short = "s")]
+    account_sequence: Option<String>,
+
+    #[options(help = "key file for the signer", short = "k")]
+    signer_key: Option<String>,
 }
 
 impl TxRawConnInitCmd {
     fn validate_options(&self, config: &Config) -> Result<ConnectionOpenInitOptions, String> {
+        // Get the account sequence
+        let parsed = self
+            .account_sequence
+            .clone()
+            .ok_or_else(|| "missing account sequence".to_string())?
+            .parse::<u64>();
+
+        let acct_seq = match parsed {
+            Ok(v) => v,
+            Err(e) => return Err("invalid account sequence number".to_string()),
+        };
+
+        // Get content of key seed file
+        let key_filename = self
+            .signer_key
+            .clone()
+            .ok_or_else(|| "missing signer key file".to_string())?;
+
+        let key_file = Path::new(&key_filename).exists();
+        if !key_file {
+            return Err("cannot find key file specified".to_string());
+        }
+
+        let key_file_contents = fs::read_to_string(key_filename)
+            .expect("Something went wrong reading the key seed file");
```

Comment:
maybe an utility function to get the key seed and seq num?

Reply:
Yes, that's a good idea, I already have code to get the sequence number. I think all these "utility" functions should be part of a separate crate, maybe something like `cosmos-rs` since this is how it's structured in the Cosmos SDK, or we could keep in the relayer for now in a separate logic.

Reply:
yes, let's address this in a different PR